### ISSUE TITLE
Windows Chef service cannot run when using a different folder with chef MSI

### ISSUE THREAD

**Issue by lbn-chef-team** [chef/chef#3952]
Hello,

When running MSI chef-client-12.4.2-1-x86.msi on a 2008R2 node, a step propose to modify the default folder (C:\opscode) to another one (for our test it was : C:\MyFolder\opscode). 

When install is finished, the service chef-client (Chef Client Service) cannot run.
The "path to executable" defined in the service is :
"C:\MyFolder\opscode\chef\embedded\bin\ruby.exe C:\opscode\chef\bin\chef-windows-service"

So it means that the MSI file doesn't consider the folder we set during install. 
We have tested without change the folder and service is running fine.

Curiously, when using ORCA to check the MSI, I found this line in the table "ServiceInstall":
C:\opscode\chef\bin\chef-windows-service (see below capture)

![orca-chef-msi](https://cloud.githubusercontent.com/assets/14806593/10048819/00d3e358-6215-11e5-9748-207d2823f705.JPG)

Could you please fix this problem ? I recommend to use a relative path in your WIX file.

Best Regards,

LBN team.

**Comment by jkeiser** [comment]
Fixed by https://github.com/chef/chef/pull/3954 . A new version should be out shortly. Thanks!

**Comment by jkeiser** [comment]
@lbn-aaio-team give 12.4.3 a try, it should be better: https://downloads.chef.io/chef-client/

**Comment by lbn-aaio-team** [comment]
Hello, 

Using chef-client-12.4.3-1-x86.msi result to the same issue.
![service](https://cloud.githubusercontent.com/assets/14806593/10073136/33771c1e-62c7-11e5-9383-f63da8e7b3b3.JPG)

Orca display the same argument on ServiceInstall MSI table:
![msiorca](https://cloud.githubusercontent.com/assets/14806593/10073170/5f24280c-62c7-11e5-9a4c-db7e47dd144c.JPG)

The problem seems to be in the packaging of the MSI for the feature : Chef Client Service
It is not able to get relative path of chef-windows-service file. Because this is hardly set in the MSI.

Regards,

LBN team

**Comment by jaym** [comment]
it looks like the incorrect variable is used https://github.com/chef/omnibus-chef/blob/master/resources/chef/msi/source.wxs.erb#L66

**Comment by jaym** [comment]
I think if you use the [chef-client cookbook](https://github.com/chef-cookbooks/chef-client) or call `chef-service-manager -a install`, it should at least unblock you until the next release

**Comment by stuartpreston** [comment]
Closing this due to age - it looks like we do honour the https://github.com/chef/chef/blob/master/omnibus/resources/chef/msi/source.wxs.erb#L159-L163

Please file another issue if this is still a problem. Thanks.
